test/upload-test-images.sh

```diff
@@ -32,6 +33,9 @@ function upload_test_images() {
   # so the resulting yaml produced is ignored.
   # We limit the number of concurrent builds (jobs) to avoid OOMs.
   ko resolve --jobs=4 ${tag_option} -RBf "${image_dir}" > /dev/null
+
+  #build and publish images from vendor directory
+  ko resolve --jobs=4 ${tag_option} -RBf "${vendor_image_dir}" > /dev/null
```

Comment:
why do we need this?

Reply:
Instead of creating a new service. Building the grpc server image from knative serving test repo.

Reply:
there are other test images as well in serving, but it only picks up the grpc-ping, how is that handled ? is that `pkgtest.ImagePath("grpc-ping")` ?
```
2020/09/09 05:03:15 Using base gcr.io/distroless/static:nonroot for knative.dev/serving/test/test_images/grpc-ping
2020/09/09 05:03:16 Building knative.dev/serving/test/test_images/grpc-ping
2020/09/09 05:04:11 Publishing gcr.io/knative-boskos-51/kclient-e2e-img/8803/grpc-ping:latest
2020/09/09 05:04:13 Published gcr.io/knative-boskos-51/kclient-e2e-img/8803/grpc-ping@sha256:f8f53534f1a74fd7e9e4df8338c55ee8c8fee29fd8aac2f03cbacdaacbdbce72
=============================
```

also, seeing a lot of vendored files, I think thats required for test images to be built? Can we avoid vendoring them and download them part of image build in test execution ?

Reply:
added "knative.dev/serving/test/test_images/grpc-ping" in hack/tools.go to bring in this service, which is bringing in other dependencies to the vendor.